src/util.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+const axios = require("axios");
+const fs = require("fs");
+const settings = require("../settings.json");
+const Json2csvParser = require("json2csv").Parser;
+const log = require("loglevel");
+/**
+ * Take a filepath to a json object of issues
+ * and a filename to save the resulting issue data,
+ * then makes api call to GitHub to retrieve current data
+ * @param {string} data
+ * @param {string} file
+ */
+async function retrieveIssues(data, file, callback) {
```

Comment:
this function takes a `callback`, let's avoid mixing use of `async/await` with callbacks. Just `return results` below, and the function that calls `retrieveIssues` should just use `async/await`

Reply:
retrieveIssues is only ever called by yargs when the command is run. Should I just call makeCSV(results, file) directly rather than passing it as a callback?

Reply:
yes, e.g.

```js
async (opts) => {
  util.makeCSV(await util.retrieveIssues(opts.repoDataFilePath, opts.issuesDataFilePath));
}
```